apps/condo/domains/meter/gql.js

```diff
@@ -34,6 +34,12 @@ const METER_READING_FILTERS_FIELDS = '{ organization address accountNumber place
 const METER_READING_FILTER_TEMPLATE_FIELDS = `{ name employee { id } fields ${METER_READING_FILTERS_FIELDS} ${COMMON_FIELDS} }`
 const MeterReadingFilterTemplate = generateGqlQueries('MeterReadingFilterTemplate', METER_READING_FILTER_TEMPLATE_FIELDS)
 
+const COMMUNAL_METER_FIELDS = `{ number numberOfTariffs installationDate commissioningDate verificationDate nextVerificationDate controlReadingsDate sealingDate isAutomatic organization { id } property { id } resource { id } b2cApp { id } b2bApp { id } meta ${COMMON_FIELDS} }`
```

Comment:
`b2cApp`? Are these meters available for residents?

Reply:
I thought b2c is also needed for embedding into the "technician's mobile application". But as it turned out, this is not necessary and will be implemented differently. Need to be removed